examples/Backtesting/custom_analytical.py

```diff
@@ -1,124 +1,110 @@
 
+# Now, we can define the scope of our optimization problem. Our goal is to optimize
+# a high-dimensional quadratic function on a bounded input domain. We first define
+# the corresponding inputs and output of the function:
 
-def sum_of_squares(*x: float) -> float:
-    """Calculate the sum of squares."""
-    res = 0
-    for y in x:
-        res += y**2
-    return res
+parameters = [
+    NumericalContinuousParameter(name=f"x_{k}", bounds=BOUNDS) for k in range(DIMENSION)
+]
+target = NumericalTarget(name="Target", mode="MIN")
 
 
-### Creating the searchspace and the objective
+# Based on the above, we construct the black-box callable to be optimized.
+# Using the {func}`~baybe.simulation.lookup.label_columns` decorator, we can easily map
+# the columns of the raw input/output arrays to our parameter and target objects, which
+# creates the required dataframe-based lookup for the optimization loop:
 
-# As we expect it to be the most common use case, we construct a purely discrete space here.
-# Details on how to adjust this for other spaces can be found in the searchspace examples.
 
-parameters = [
-    NumericalDiscreteParameter(
-        name=f"x_{k+1}",
-        values=list(np.linspace(*BOUNDS[k], POINTS_PER_DIM)),
-        tolerance=0.01,
-    )
-    for k in range(DIMENSION)
-]
+@label_columns([p.name for p in parameters], [target.name])
+def sum_of_squares(x: np.ndarray, /) -> np.ndarray:
+    """Calculate the sum of squares."""
+    return (x**2).sum(axis=1, keepdims=True)
 
-searchspace = SearchSpace.from_product(parameters=parameters)
-objective = SingleTargetObjective(target=NumericalTarget(name="Target", mode="MIN"))
 
-### Constructing campaigns for the simulation loop
+# What remains is to construct the search space and objective for the optimization:
 
-# To simplify adjusting the example for other recommenders, we construct some recommender objects.
-# For details on recommender objects, we refer to [`recommenders`](./../Basics/recommenders.md).
+searchspace = SearchSpace.from_product(parameters=parameters)
+objective = target.to_objective()
 
-seq_greedy_EI_recommender = TwoPhaseMetaRecommender(
-    recommender=BotorchRecommender(acquisition_function="qEI"),
-)
-random_recommender = TwoPhaseMetaRecommender(recommender=RandomRecommender())
+### Creating the Campaigns
 
-# We now create one campaign per recommender.
+# We consider two optimization scenarios, each represented by its own campaign:
+# * Optimization using the default recommender
+# * A baseline using randomly generated recommendations
 
-seq_greedy_EI_campaign = Campaign(
+default_campaign = Campaign(
     searchspace=searchspace,
-    recommender=seq_greedy_EI_recommender,
     objective=objective,
 )
 random_campaign = Campaign(
     searchspace=searchspace,
-    recommender=random_recommender,
     objective=objective,
+    recommender=RandomRecommender(),
 )
 
-### Performing the simulation loop
+### Running the Optimization Loop
```

Comment:
this should remain `Simulation` because this is a MC based simulation experiment and not a no-MC optimzization

Reply:
I don't see the problem 🤔 Each MC iteration does perform an actual optimization loop. So what is the issue? I can also change, but then probably need to change the entire example text because "optimization" appears several times ...

Reply:
if you prefer it, then I'm OK with it. Changed